Sources/Datadog/Datadog.swift

```diff
@@ -125,6 +125,14 @@ public class Datadog {
         return CoreRegistry.default is DatadogCore
     }
 
+    /// Returns the core instance for the given name.
+    ///
+    /// - Parameter name: The name of the instance to get.
+    /// - Returns: The core instance if it exists, `NOPDatadogCore` instance otherwise.
+    public static func coreInstance(named name: String) -> DatadogCoreProtocol {
```

Comment:
question/ What is the counterpart of this API in `dd-sdk-android`?

Reply:
It's [`getInstance`](https://github.com/DataDog/dd-sdk-android/blob/feature/sdkv2/dd-sdk-android-core/src/main/kotlin/com/datadog/android/Datadog.kt#L138) on Android. I think this version is more explicit and swift-friendly, WDYT?